src/views/NotFound.tsx

```diff
@@ -0,0 +1,20 @@
+
+function notfound() {
+  return (
+    <div className="w-screen flex h-screen bg-late-white justify-center items-center">
+      <div className="w-4/5 sm:w-5/6 sm:h-4/5 h-2/3 bg-coffee-brown rounded-xl flex sm:flex-col p-10">
+        <div className="flex-1 justify-evenly  flex text-white flex-col gap-10">
+          <div className="flex gap-16 sm:gap-3 flex-col">
+            <h1 className="text-9xl playfair sm:text-5xl">Oops!</h1>
```

Comment:
don't right hard string in this function , right it in en.json file and use the i18n